n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -145,7 +145,13 @@ def from_yaml(cls, o, *args, **kwargs):
             validation_action = namespace + "validation_pseudodata"
 
             N3FIT_FIXED_CONFIG['actions_'].extend((training_action, validation_action))
-
+        N3FIT_FIXED_CONFIG['theory_covmat_flag'] = False
+        N3FIT_FIXED_CONFIG['use_user_uncertainties'] = None
+        N3FIT_FIXED_CONFIG['use_scalevar_uncertainties'] = None
```

Comment:
Is there a better way to set these defaults validphys-wise @Zaharid ?

Reply:
Maybe use production rules. But this is fine really.